sdk/attestation/attestation/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 Microsoft
```

Comment:
I'm no copyright expert, but should this be updated (and all our licenses) to 2021? @bterlson

Reply:
I thought the official MSFT copyright notice was:

```
Copyright (c) Microsoft Corp. All Rights Reserved
```

But my knowledge may be out of date.